packages/serverpod_client/lib/src/serverpod_client_browser.dart

```diff
@@ -32,14 +32,16 @@ class ServerpodClientRequestDelegateImpl
   Future<String> serverRequest<T>(
     Uri url, {
     required String body,
+    String? authenticationValue,
   }) async {
     try {
-      var response = await _httpClient
-          .post(
-            url,
-            body: body,
-          )
-          .timeout(connectionTimeout);
+      var response = await _httpClient.post(
+        url,
+        body: body,
+        headers: {
+          if (authenticationValue != null) 'authorization': authenticationValue,
```

Comment:
Question: While HTTP headers are not case sensitive, I think it's most common to use capitalized values (i.e. `Authorization`). Could be worth considering to be as similar to the standard case as possible?

Reply:
Good point!